src/components/NormalLeftRow/index.jsx

```diff
@@ -28,20 +28,39 @@ function NormalLeftRow({ schema, classes, indent }) {
   const name = 'name' in schema ? schema.name : null;
   /**
    * Define the type symbol for the schema or sub-schema's type
-   * Types requiring nested structures use the according bracket symbol
+   * Types requiring nested structures use the according bracket symbol,
+   * Complex types (allOf, anyOf, oneOf, no) use comment notation,
    * while others use highlighted text to illustrate the data type.
    */
   const bracketTypes = ['array', 'object', 'closeArray', 'closeObject'];
-  const typeSymbol = bracketTypes.includes(schema.type) ? (
-    {
-      array: '[',
-      object: '{',
-      closeArray: ']',
-      closeObject: '}',
-    }[schema.type]
-  ) : (
-    <code className={classes.code}>{schema.type}</code>
-  );
+  const combinationTypes = [
+    'allOf',
+    'anyOf',
+    'oneOf',
+    'not',
+    'and',
+    'or',
+    'nor',
+  ];
+  const typeSymbol =
+    bracketTypes.includes(schema.type) ||
+    combinationTypes.includes(schema.type) ? (
+      {
+        array: '[',
+        object: '{',
+        closeArray: ']',
+        closeObject: '}',
+        allOf: '// All of',
+        anyOf: '// Any of',
+        oneOf: '// One of',
+        not: '// Not',
+        and: '// and',
+        or: '// or',
+        nor: '// nor',
+      }[schema.type]
```

Comment:
For the comments, what do you think about the following style:

```jsx
color: theme.palette.text.hint,
```

#### Before
![Screen Shot 2020-01-03 at 2 28 57 PM](https://user-images.githubusercontent.com/3766511/71744524-89e74180-2e35-11ea-892f-f4014f528df5.png)

#### After
![Screen Shot 2020-01-03 at 2 28 45 PM](https://user-images.githubusercontent.com/3766511/71744525-8a7fd800-2e35-11ea-89f4-88a561ab3529.png)

Reply:
Great idea 👍  Applied changes!

Reply:
Great idea 👍 Applied changes!